timemachine/fe/single_topology_v3.py

```diff
@@ -779,34 +995,71 @@ def setup_intermediate_state(self, lamb):
         """
         src_system = self.src_system
         dst_system = self.dst_system
-        interpolate_fn = interpolate.linear_interpolation
 
-        # tbd: use different interpolation functions later
+        # alchemical parameter values at which angle and torsion force constants transition from zero to nonzero for
+        # ring closing transformations
+        lambda_angles = 0.4  # for angles, src_k = 0 => k(lamb) = 0 when lamb < lambda_angles
```

Comment:
Pretty sure I understand what this is getting at, but not really clear on how to read this syntax
```suggestion
        lambda_angles = 0.4  # angle param interpolation for lamb < lambda_angles = 0
```

Reply:
Hopefully this is clarified by the docstring addition in https://github.com/proteneer/timemachine/pull/820/commits/300377099ccf6981338d5c196bc508fc3289f815